package.json

```diff
@@ -105,9 +106,11 @@
     ]
   },
   "devDependencies": {
+    "@types/node": "^6.0.36",
```

Comment:
Is this doing anything? In the generated project I had to specify `typeRoots` for it to work from a subdir.